assets/src/dashboard/components/storyMenu/index.js

```diff
@@ -114,11 +126,14 @@ export default function StoryMenu({
 }
 
 StoryMenu.propTypes = {
+  isInverted: PropTypes.bool,
   itemActive: PropTypes.bool,
   tabIndex: PropTypes.number,
   storyId: PropTypes.number,
   onMoreButtonSelected: PropTypes.func.isRequired,
   contextMenuId: PropTypes.number.isRequired,
   menuItems: PropTypes.arrayOf(PropTypes.shape(MenuItemProps)).isRequired,
+  menuLabel: PropTypes.string,
+  menuStyleOverrides: PropTypes.array,
```

Comment:
is this supposed to be an object?

Reply:
it's actually an array because it's `{ css }`
> Returns an array of interpolations, which is a flattened data structure that you can pass as an interpolation itself.
https://styled-components.com/docs/api#css

Reply:
whoaaa TIL. nice!